packages/loaders/src/Progress.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const COMPONENT_ID = 'loaders.progress';
+
+const { zdColorGreen600, zdColorGrey200, zdSpacingXs } = require('@zendeskgarden/css-variables');
```

Comment:
This should be a standard `import {} from '..'` structure rather than the `require` syntax.